### DISCUSSION TITLE
Mapas en formato circular

### DISCUSSION THREAD

**Discussion by JamirGarro** [qgispe/discusiones#20] [Preguntas&Respuestas] [answered]
Hola buen día, como podria hacer mapas de forma circular, he explorado casi todas las propiedades y no encontre alguna opcion que le diera forma circular al nuevo mapa en la composición.

Adjunto imagen del resultado que espero obtener:
![imagen_2021-07-22_123612](https://user-images.githubusercontent.com/86505835/126683613-76f007f0-b516-4089-9ac3-bbc2b247769e.png)

**Comment by ambarja** [comment, marked as answer]
Hola @JamirGarro existen varias formas desde plugins hasta el mismo diseñador de mapas , aquí te adjunto algunas alternativas, una opción es usando dentro del compositor de mapas > formas geométricas >  luego hacer un clip y listo, otra alternativa es usando el generador de geometrías como  `make_circle`.

![s](https://user-images.githubusercontent.com/23284899/126699702-7f8c24ca-3811-4487-8ed4-9ed3aae6cf6e.gif)

**Reply by JamirGarro** [reply]
hola antonhy que version de qgis es esa? yo cuento con la version 3.10 y no me aparece la opción clipping setings.
no se si me podrias mencionar cuales son esos plugins que me ayudarian a solucionar este inconveniente
![imagen_2021-07-22_170642](https://user-images.githubusercontent.com/86505835/126715487-e553314f-94e5-46ca-96d7-c1a0fff928ba.png)

adjunto imagen de lo mencionado.

**Reply by ambarja** [reply]
Hola @JamirGarro  desde la versión 3.16 la tienes disponible, por otro lado también te sugiero seguir los pasos que menciona @lefcgis  si tienes versiones inferiores :)

**Reply by JamirGarro** [reply]
okey anthony ya logre solucionarlo creando una elipse en forma de circulo y luego en configuraciones avanzadas le aplique un iluminado tipo punto de 500 y luego en renderizado le di iluminado, de esta manera tranparentaba a mi mapa de abajo creandole un marco circular
![imagen_2021-07-22_175423](https://user-images.githubusercontent.com/86505835/126719034-7a86039c-1512-404c-b8fa-aba9ec084a64.png)
.

**Reply by ambarja** [reply]
Excelente @JamirGarro  ^^ #qgispe

**Comment by lefcgis** [comment]
Hola @JamirGarro . Complementando lo que menciona @ambarja . puedes también crear una capa "temporal" a nivel puntual y añadir un punto aleatorio en alguna parte de tu vector principal, en tu caso sería el del polígono verde. De lo contrario, podría utilizar la herramienta "Punto en superficie y declaras que la capa de entrada sería la de tu polígono verde. Luego, utilizarías el algoritmo "buffer" o área de influencia y le das, por ejemplo un radio específico (5 kilómetros de distancia). Configuras tus parámetros de entrada para el geoproceso y te sugiero que le asignes un total de 20 segmentos como mínimo, para que tu vector verde se acerque lo más posible a tu nueva capa buffer. Como ejemplo tienes este [tweet](https://twitter.com/quecrees/status/1401501063799316481/photo/1) 😄 Escríbeme cualquier cosa y esperamos haberte podido ayudar.

**Reply by JamirGarro** [reply]
hola si, esta fue la primerasolucion que se me ocurrio pero con el buffer no me quedaba tan estetico como esperaba, por eso buscaba una misma solucion en el mismo compositor, algo asi como en arcgis.
pero ya pude solucionarlo, gracias :).
